helpers/kube/kube.go

```diff
@@ -37,14 +52,24 @@ func GetKubeClient() *kubernetes.Clientset {
 
 // GetPolicyClient creates a policy clientset
 func GetPolicyClient() (*policy.Client, error) {
-	// Get configuration
-	cfg, err := rest.InClusterConfig()
+	var config *rest.Config
+	var err error
+
+	// If KUBECONFIG ENV var is set, use that kubeconfig file location to create the kube client
+	kubeconfig, kubeconfigSet := os.LookupEnv("KUBECONFIG")
+	if kubeconfigSet {
+		glog.Info(fmt.Sprintf("KUBECONFIG env variable is set to %s, using this for kube client config", kubeconfig))
+		config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
+	} else {
+		glog.Info("KUBECONFIG env variable is NOT set, defaulting to in-cluster kube client config")
+		config, err = rest.InClusterConfig()
+	}
 	if err != nil {
-		return nil, err
+		glog.Fatal(err)
```

Comment:
I think this line could stay as `return nil, err` 
The [upstream code](https://github.com/IBM/portieris/blob/master/cmd/trust/main.go#L63) checks for the err, and will fatal accordingly.

Reply:
ohhh got it, will fix.  thanks.